CADETProcess/optimization/optimizationProblem.py

```diff
@@ -589,32 +590,37 @@ def get_dependent_values(self, x):
 
         Returns
         -------
-        x : np.ndarray
+        np.ndarray
             Value of all optimization variables in untransformed space.
 
         """
```

Comment:
Maybe catch the error that variables are given as a 1d array, or is this happening in ensures2d? Just checked :+1:

Reply:
Yeah, `ensures2d` automatically converts everything to an array with ndim=2 so this method should never see a 1d array.